test/ExpressiveInstallerTest/InstallerTestCase.php

```diff
@@ -141,18 +140,33 @@ protected function getContainer()
         return $this->container;
     }
 
-    protected function getAppResponse($path = '/')
+    protected function getAppResponse($path = '/', $setupRoutes = true)
     {
         $container = $this->getContainer();
 
         /** @var Application $app */
-        $app     = $container->get('Zend\Expressive\Application');
-        $request = new ServerRequest([], [], 'https://example.com' . $path, 'GET');
+        $app = $container->get(Application::class);
+
+        // Import programmatic/declarative middleware pipeline and routing configuration statements
+        $app->pipe(\Zend\Stratigility\Middleware\ErrorHandler::class);
+        $app->pipe(\Zend\Expressive\Helper\ServerUrlMiddleware::class);
+        $app->pipeRoutingMiddleware();
+        $app->pipe(\Zend\Expressive\Helper\UrlHelperMiddleware::class);
```

Comment:
When you update the default pipeline file as noted in a previous comment, this test will need to change to reflect the additions.